core/src/main/resources/lib/form/select/select.js

```diff
@@ -5,9 +5,14 @@ function updateListBox(listBox,url,config) {
     config = object(config);
     var originalOnSuccess = config.onSuccess;
     var l = $(listBox);
-    var status = findFollowingTR(listBox, "validation-error-area").firstChild.nextSibling;
-    if (status.firstChild && status.firstChild.getAttribute('data-select-ajax-error')) {
-        status.innerHTML = "";
+    var status;
+    try {
+        status = findFollowingTR(listBox, "validation-error-area").firstChild.nextSibling;
+        if (status.firstChild && status.firstChild.getAttribute('data-select-ajax-error'))
+            status.innerHTML = "";
+    } 
+    catch(e) {
+        console.log(e);
```

Comment:
If it's a critical error it should probably be printed out with a `console.error` or `console.warn`